contrib/segmentio/kafka-go/option.go

```diff
@@ -7,15 +7,6 @@ package kafka
 
 import "github.com/DataDog/dd-trace-go/contrib/segmentio/kafka-go/v2/internal/tracing"
 
-type config struct {
```

Comment:
This came up while merging. It seems that this isn't being used anywhere in v2, but I would like to double check it.